### DISCUSSION TITLE
"Used by" list

### DISCUSSION THREAD

**Discussion by arkivanov** [arkivanov/Decompose#366] [Show and tell]
It would be good to have a list of companies/users/projects where Decompose is used. I suggest to stick to the following format, but feel free to omit/add/modify any details. ❤️

Project name: \<project name\>
Project type: production/pet/sample
Company name: personal/\<company name\>
Supported platforms: Android, JVM, Web, iOS, macOS, etc.
Link (an open source repository or a web site): \<link\>

**Comment by moffpage** [comment]
Project name: Vlife
Description: Vlife is an app for drivers and everyone who likes to get Points and privileges for their usual purchases: sign up for a car wash, make purchases in installments in Market, pay for utilities, take out insurance and accumulate Points with our partners.
Company Name: LLC "GRAND ERA"
Project type: Production
Supported targets: currently only Android
You can install in the app Google Play Store: https://play.google.com/store/apps/details?id=kz.vlife.grandera

**Comment by a-blekot** [comment]
Project name: Memorize shlokas (Sanskrit verses study)
Project type: production
Company name: personal (Aleksey Blekot)
Supported platforms: Android, iOS, Mac M1 😊
[PlayStore](http://play.google.com/store/apps/details?id=com.a_blekot.memorize_shloka)
[AppStore](https://apps.apple.com/app/id6443863948)
[GitHub](https://github.com/a-blekot/memorize_shloka)

**Comment by egorikftp** [comment]
<img src="https://github.com/user-attachments/assets/808c13c2-b897-47f0-873d-92be100d7383" width="150" />


Project name: За рулем (Гродно)
Description: Application helps to observe actual situation on the roads
Project type: production
Company name: personal
Supported platforms: Android, iOS, Mac M1

[Google Play](https://play.google.com/store/apps/details?id=com.egoriku.grodnoroads) 
[AppStore](https://apps.apple.com/app/id6737742923)
[Github](https://github.com/BehindWheel/BehindWheelKMP/)

**Comment by brandonpollack23** [comment]
Project name: Before You Die
Project type: Personal Task Managment/ TODO list software with blocking/blocked by feature
Company name: personal
Supported platforms: Android, JVM, Web, (iOS when UI is done).
Link (an open source repository or a web site): https://github.com/brandonpollack23/before_you_die

**Comment by ashdavies** [comment]
Project name: Playground
Project type: pet
Platforms: Android, JVM, Server
Link: [ashdavies/playground](https://github.com/ashdavies/playground)

**Comment by oianmol** [comment]
Project name: Slack Compose Kotlin Multiplatform Clone with gRPC
Project type: pet
Company name: personal
Supported platforms: Android, JVM, iOS
Link : https://github.com/oianmol/SlackComposeMultiplatform

**Comment by azlekov** [comment]
Project name: Dodi
Description: Simplify access management of short-term rentals
Project type: pet
Company name: DODI Smart
Supported platforms: Android, iOS
Link: https://dodi.li

**Comment by t1r** [comment]
Project name: Compose Theme Builder
Project type: pet
Company name: personal
Supported platforms: Android, JVM, (iOS in future)
Link: [ComposeThemeBuilder](https://github.com/t1r/ComposeThemeBuilder)

**Comment by blvckcoded** [comment]
Project name: DodoForMastodon
Project type: production/pet
Company name: open source, led by Androiddev-social community
Supported platforms: Android, iOS, and Desktop JVM
Link: https://github.com/AndroidDev-social/DodoForMastodon

**Comment by xxfast** [comment]
Project name: NYTimes-KMP
Project type: sample
Company name: personal
Supported platforms: Android, JVM, iOS, and Web
Link: https://github.com/xxfast/NYTimes-KMP

**Comment by MohamedRejeb** [comment]
Project name: Pokedex
Project type: sample
Company name: personal
Supported platforms: Android, JVM, iOS
Link: https://github.com/M0Coding/Pokedex

**Comment by arkivanov** [comment]
Project name: Confetti
Project type: production/sample
Author: https://github.com/joreilly
Supported platforms: Android, iOS
Link: https://github.com/joreilly/Confetti

**Comment by arkivanov** [comment]
Project name: Sample TodoApp
Project type: sample
Original author: arkivanov
Supported platforms: Android, iOS, Web
Link: https://github.com/IlyaGulya/TodoAppDecomposeMviKotlin
Stack: Decompose, MVIKotlin, SQLDelight, Compose, SwiftUI

**Comment by malliaridis** [comment]
Project name: OBSSD Premium
Description: A remote stream deck for OBS Studio and Streamlabs OBS
Project type: production
Company name: personal
Supported platforms: Android (app), Multiplatform (APIs and shared logic)
Link: [Google Play Store - OBSSD Premium](https://play.google.com/store/apps/details?id=com.malliaridis.obs.android)

**Comment by alaershov** [comment]
Project name: Whoosh
Description: Electric scooter sharing app
Project type: production
Company name: Whoosh
Supported platforms: Android
Link: [Google Play - Whoosh](https://play.google.com/store/apps/details?id=com.punicapp.whoosh)

**Comment by YektaDev** [comment]
Project name: **Yaad**
Description: **A knowledge management system for everyday use.**
Project type: **Production**
Company name: **Personal**
Supported platforms: **Android, JVM**
Link: **[Yaad.One](https://yaad.one)**

**Comment by matejsemancik** [comment]
Project name: **Loono**
Description:  **Loono is your personal guide to prevention, helping you lead a healthier, higher-quality life, showing you the right way to care for your health.**
Project type: **production**
Company name: Loono / developed by @futuredapp
Supported platforms: **Android, iOS**
Link:
https://www.loono.cz/en/mobile-app
[Google Play](https://play.google.com/store/apps/details?id=cz.loono.app)
[Apple AppStore](https://apps.apple.com/cz/app/loono/id1573646003?platform=iphone)

(this was originally Flutter app which we rebuilt from scratch in KMP using Decompose, recently released)

**Comment by thomaskioko** [comment]
Project name: TvManiac
Project type: pet
Supported platforms: Android, iOS
Link: https://github.com/thomaskioko/tv-maniac

**Comment by SEAbdulbasit** [comment]
Project name: MusicApp-KMP
Project type: pet
Supported platforms: Android, iOS, Web, Desktop
Link: https://github.com/SEAbdulbasit/MusicApp-KMP

**Comment by avan1235** [comment]
<img src="https://github.com/arkivanov/Decompose/assets/11787040/58edb0d7-eec7-45f2-85c2-07602ffab137" 
     width="100" 
     height="100" />
Project name: OpenOTP
Project type: pet (published on [Google Play](https://play.google.com/store/apps/details?id=ml.dev.kotlin.openotp.OpenOtp))
Project description: Kotlin Multiplatform implementation of One Time Passwords Manager
Company name: personal
Supported platforms: Android, iOS, Desktop
Link: https://github.com/avan1235/open-otp

**Comment by avan1235** [comment]
<img src="https://github.com/arkivanov/Decompose/assets/11787040/1664682c-5fc3-46fe-bc73-1b3e9e638361" 
     width="100" 
     height="100" />
Project name: MiniGames
Project type: pet (published on [Google Play](https://play.google.com/store/apps/details?id=ml.dev.kotlin.minigames))
Project description: Online games written in pure Compose Multiplatform
Company name: personal
Supported platforms: Android, iOS, Desktop, Wasm
Link: https://github.com/avan1235/mini-games/

**Comment by avan1235** [comment]
<img src="https://github.com/arkivanov/Decompose/assets/11787040/b3c853b6-5e31-4b86-80d0-304b8eca3f58" 
     width="100" 
     height="100" />
Project name: Shin
Project type: pet
Project description: URL Shortener in Compose Multiplatform
Company name: personal
Supported platforms: Android, iOS, Desktop, Wasm
Link: https://github.com/avan1235/shin/

**Comment by AJIEKCX** [comment]
Project name: SpaceXRockets
Project type: sample
Company name: personal
Supported platforms: Android, iOS
[GitHub](https://github.com/AJIEKCX/SpaceXRockets)

**Comment by AJIEKCX** [comment]
![image](https://github.com/arkivanov/Decompose/assets/18354891/0abd5f03-5bb7-4bab-96cc-36731602d9fb)
Project name: Kontur.Market
Project type: production
Company name: Kontur
Supported platforms: Android, iOS
[Google Play](https://play.google.com/store/apps/details?id=ru.kontur.market), [App Store](https://apps.apple.com/ru/app/%D0%BA%D0%BE%D0%BD%D1%82%D1%83%D1%80-%D0%BC%D0%B0%D1%80%D0%BA%D0%B5%D1%82/id6446759025)

**Comment by giorgioprovenzale** [comment]
Project name: NewsApp
Project type: sample
Company name: personal
Supported platforms: Android, iOS
Link: https://github.com/giorgioprovenzale/KmpNewsApp

**Comment by YektaDev** [comment]
<img width="128" height="128" alt="Logo" src="https://github.com/arkivanov/Decompose/assets/22149430/649644e8-986d-49e6-9706-6c69e78d61e1" />

Project name: **English Quotes Pro**
Description: **A collection of quotes in various categories.**
Project type: **Pet**
Company name: **Personal**
Supported platforms: **Android**
Link: **https://yekta.dev/creations/#english-quotes-pro**

**Comment by YektaDev** [comment]
<img width="128" height="128" alt="Logo" src="https://github.com/arkivanov/Decompose/assets/22149430/bf9ef8b2-82d9-4557-98d6-0fdf9188ca6f" />

Project name: **Super Tic Tac Toe**
Description: **Tic-tac-toe inside tic-tac-toe!**
Project type: **Pet**
Company name: **Personal**
Supported platforms: **Android**
Link: **https://yekta.dev/creations/#super-tic-tac-toe**

**Comment by kasem-sm** [comment]
![image](https://github.com/user-attachments/assets/f668578d-f4ac-4708-9441-3ffa0bb4e0ae)

Project name: PaletteAI
Project type: production
Company name: personal/The True Studio
Supported platforms: Android, iOS
Link: [App Store](https://apps.apple.com/us/app/paletteai-dark-4k-wallpapers/id6471511124), [Google Play Store](https://play.google.com/store/apps/details?id=truestudio.ai)

**Comment by T8RIN** [comment]
![logo](https://github.com/user-attachments/assets/4981f631-7fd2-43cd-8eab-f1f53d05523c)
Project name: Image Toolbox
Project type: production
Company name: personal
Supported platforms: Android
Link (an open source repository or a web site): [Github](https://github.com/T8RIN/ImageToolbox), [Google Play](https://play.google.com/store/apps/details?id=ru.tech.imageresizershrinker)

**Comment by VladislavSumin** [comment]
<img
        src="https://github.com/user-attachments/assets/9ab67ee8-85e9-404d-8f32-367d83f48eb6" 
        width=18%
        title="Dzen logo"
        alt="Dzen logo"
    />

Project name: Dzen
Project type: production
Company name: VK
Supported platforms: Android
Link (an open source repository or a web site): [Site](https://dzen.ru/), [RuStore](https://www.rustore.ru/catalog/app/ru.zen.android), [Google Play](https://play.google.com/store/apps/details?id=ru.zen.android)

**Comment by arkivanov** [comment]
<img src="https://github.com/user-attachments/assets/365bdcde-d231-4812-b59f-17b04c0dc3f1" width=128 title="X logo" alt="X logo" />

Project name: 𝕏 (formerly Twitter)
Project type: production
Company name: 𝕏
Supported platforms: Android
Link: [Google Play](https://play.google.com/store/apps/details?id=com.twitter.android), [x.com](https://x.com)

**Reply by a-blekot** [reply]
Hi @arkivanov, have you migrated X to Decompose?

**Reply by arkivanov** [reply]
The new [Android app rewrite](https://x.com/FredLohner/status/1920942018202923504) is 100% Decompose. Some of its features are pushed to the main X app.

**Comment by adrianegraphene** [comment]
![logo_karmacall_highres_transparent-trans.png](https://github.com/user-attachments/assets/1727962c-f894-4732-83de-4ef2e1a53fe0)


Project name: KarmaCall
Project type: production
Company name: FynCom / RoboCash Inc.
Supported platforms: Android
Link: [Website](https://www.karmacall.com/),  [Google Play](Https://play.google.com/store/apps/details?id=com.fyncom.robocash),  [iOS](Https://apps.apple.com/us/app/karmacall/id1574524278)

**Comment by MarketTwits** [comment]
![image](https://github.com/user-attachments/assets/bcf3f58b-aaac-4fd1-a426-d882cfc9baa2)

Project name: Sportsauce 🏃‍♂️‍➡️
Project type: production
Company name: Sports Union (Sportsauce)
Supported platforms: Android, JVM, JS
Link: [Google Play](https://play.google.com/store/apps/details?id=com.markettwits.sportsouce), [Website](https://sportsauce.ru/), [GitHub](https://github.com/MarketTwits/Sportsouce)

**Comment by bartwell** [comment]
Project name: Delight SQL Viewer
Project type: production
Company name: personal
Supported platforms: Android, iOS, JVM
Link: https://github.com/bartwell/delight-sql-viewer

**Comment by sunildhiman90** [comment]
Project name: KMPEcommerceProductsApp
Project type: sample
Company name: personal
Supported platforms: Android, iOS, Desktop & Web(Kotlin/Js)
Link: https://github.com/sunildhiman90/Kmp-App-Shared-Ui-Compose-Multiplatform-App

**Comment by soniccat** [comment]
![ic_launcher-playstore](https://github.com/user-attachments/assets/5ebe0c7b-8adc-465a-9fb8-08880eeaed96)

Project name: Word Teacher - an app to help learning new english words
Project type: pet
Company name: personal
Supported platforms: Android, JVM
Link: https://github.com/soniccat/WordTeacher

**Reply by a-blekot** [reply]
hey @soniccat, is it available on PlayStore? or should I built it myself + setup BE?

**Reply by soniccat** [reply]
@a-blekot hey. I am afraid, but it isn't available in PlayStore. However you can find it in other ones. The full list of the stores is here https://github.com/soniccat/WordTeacher/releases/tag/1.8. Also you can just download the apk attached there at the bottom of the page.

**Reply by a-blekot** [reply]
Great) Thank you so much!

**Comment by heldercorreia** [comment]
Project name: Tunity - a free app to hear live audio from muted televisions
Project type: production
Company name: iSpot
Supported platforms: Android, iOS
Link: https://www.tunity.com/#download

**Comment by bartwell** [comment]
Project name: Kick — Kotlin Inspection & Control Kit
Project type: production
Company name: personal
Supported platforms: Android, iOS, JVM
Link: https://github.com/bartwell/kick

**Comment by CXwudi** [comment]
Project name: Realworld Conduit Example App
Project type: Sample, practice
Company name: Personal
Supported platforms: Android, JVM, Web
Link: https://github.com/CXwudi/realworld-cmp-decompose-mvikotlin-app

This is a frontend implementation of the [RealWorld](https://github.com/gothinkster/realworld) repository. The project was initially a practice to learn Compose Multiplatform development. Overall, I really enjoyed the development and the knowledge I gained along the way, especially regarding Decompose and MVIKotlin. Despite their complexity, they effectively address underlying problems that many developers, and even other navigation/MVI libraries, might not be aware of.

**Comment by yet300** [comment]
Project name - Tetris Lite 
Company name - Personal 
Link - https://github.com/yet300/TetrisLite
Supported platform - Android, IOS, Desktop, MacOS, JS
TetrisLite - KMP Tetris for All Platforms 

Libs: Decompose(CMP and Swift UI), MVI Kotlin

**Comment by amir1376** [comment]
Project name: AB Download Manager
Project type: production
Company name: personal
Supported platforms: Windows, macOS, Linux, Android
Link: [Website](https://abdownloadmanager.com), [GitHub](https://github.com/amir1376/ab-download-manager)

**Comment by 3moly** [comment]
<img width="100" height="100" alt="cedar" src="https://github.com/user-attachments/assets/06f55f06-ecb7-44bf-a177-4ff27d15db65" />

Project name: CedarJam - Note-Taking App
Project type: pet
Company name: personal
Supported platforms: Windows, macOS, Linux, Android, IOS
Link: https://github.com/3moly/CedarJam

**Comment by trykovyura** [comment]
Project name: Surprise Santa - Gift Exchange
Project type: production
Company name: personal
Author: https://github.com/yet300
Supported platforms: Android, iOS
Link: [Google Play](https://play.google.com/store/apps/details?id=ru.trykov.secretsanta), [App Store](https://apps.apple.com/us/app/surprise-santa-gift-exchange/id1185304926), [Website](https://surprisesantas.com/en)
Stack: Decompose, MVIKotlin, Compose, KtorFit, Koin

**Comment by Wavesonics** [comment]
<img src="https://raw.githubusercontent.com/Wavesonics/hammer-editor/refs/heads/develop/readme/logo.svg" width="96" alt="Hammer Logo" />

**Project name:** Hammer
**Project type:** production
**Company name:** Dark Rock Studios
**Author:** https://github.com/Wavesonics
**Supported platforms:** Windows, macOS, Linux, Android, iOS, Server
**Link:** [Github](https://github.com/Wavesonics/hammer-editor), [Website](https://hammer.ink/), [Google Play](https://play.google.com/store/apps/details?id=com.darkrockstudios.apps.hammer.android), [App Store](https://apps.apple.com/us/app/hammer-editor/id6771650681), [Other](https://hammer.ink#download)
**Stack:** Decompose, Compose, Koin, Ktor

**Comment by behzodhalil** [comment]
**Project name**: Better - Gym & Calorie Tracker
**Project type**: production
**Company name**: personal
**Author**: https://github.com/behzodhalil
**Supported platform**s: Android, iOS
**Link**: [Google Play](https://play.google.com/store/apps/details?id=io.behzodhalil.better), [App Store](https://apps.apple.com/us/app/better-gym-calorie-tracker/id6769016777),

**Comment by xxfast** [comment]
<img width="180" src="https://play-lh.googleusercontent.com/N_Wb8tZ2JYU2O-gWEIE3N87qagJ5_pcCScbqCODxUpZr8wu8jRcDNpgoGuOJg_9sX-Qpv04Qu2bbWU8yEUc=w480-h960-rw"/>

Project name: PSCore
Project type: Production
Company name: Motorola Solutions
Supported platforms: Android, iOS
Link: [Google Play](https://play.google.com/store/apps/details?id=au.com.motorolasolutions.pscore&hl=en_AU), [App Store](https://apps.apple.com/au/app/pscore/id6755268423)
